config/locales/en.yml

```diff
@@ -556,8 +556,8 @@ en:
   sites:
     spam_protections:
       edit:
-        captcha_hint_false: 'reCAPTCHA has not been configured correctly, Spam Protection cannot be enabled.'
-        captcha_hint_true: "When 'Suspicious only' is selected then reCAPTCHA is shown only if the automated checks detect a possible spammer. For 'Always', reCAPTCHA will always appear when a form is presented to a user who is not logged in."
+        captcha_hint_false: 'reCAPTCHA has not been configured correctly, Bot Protection cannot be enabled.'
+        captcha_hint_true: "reCAPTCHA will be checked also when the user is logged in."
```

Comment:
Hmm... is this true though? :sweat_smile: 

The protection is currently enabled on the Signup and Forgot password pages, right? It's where the user is not logged in.

Maybe we can specify here that reCAPTCHA v3 will invisibly verify interactions to detect bots.

Reply:
> Hmm... is this true though? 😅

Well, it's technically true because we had a restriction and captcha was only being checked when the user wasn't logged in, but I removed that restriction, so now captcha is checked on any screen you place it. A possible future use case could be the message composing screen. However it's true that, right now, the captcha is only present on screens where the user isn't logged in.

TBH, I added that message because I needed a `captcha_hint_true` for this line:

https://github.com/3scale/porta/blob/8af0df83d9d386b559053d04ea8e366cfe1d4b51/app/views/sites/spam_protections/edit.html.erb#L11

And I didn't know what to say there really. Your hint about invisible interactions fits good, though. So I pushed it (https://github.com/3scale/porta/pull/3645/commits/08bf84a99ddc74eb282833e7a1de0b6dbc538132).

Reply:
On pages where user is not expected to be logged in, then no need to check whether they are logged in, because even if for some reason they count as logged-in, they shouldn't be.

And if we want captcha on a page where user is supposed to be logged in, then we don't need that check either.

In both cases the better option is to ignore logged-in status.